Framework/PythonInterface/mantid/__init__.py

```diff
@@ -49,7 +49,11 @@ def _bin_dirs():
     # standard packaged install
     yield _moduledir
 
-    # conda layout
+    # conda environment
+    if 'CONDA_PREFIX' in os.environ:
```

Comment:
Does this not match the `yield os.path.dirname(sys.executable)` below?

Reply:
No it finds the base python. We need this for working inside conda environments.

Reply:
Sorry I think I misunderstanding something.
```bash
$ conda activate mantidsandbox
$ echo $CONDA_PREFIX
/Users/dmn58364/opt/miniconda3/envs/mantidsandbox
$ python -c 'import sys; print(sys.executable)'
/Users/dmn58364/opt/miniconda3/envs/mantidsandbox/bin/python
```
that are then the same? I thought `sys.executable` was always the executable that was running so would be by definition the correct bin directory?

Reply:
I would agree, but it was definitely coming out wrong for me. Try reverting the changes I made in the installed `mantid/__init__.py` and seeing if you can get it to start up.

Reply:
I just tried a fresh build of the latest changes from conda-recipes and changed the SHA to be the HEAD of this branch.
The package does seem to work correctly for me if I comment out these lines. Shall I send you my package?

Reply:
Yes please